README.md

```diff
@@ -4,6 +4,6 @@
 
 Paste the following code into your browser address bar:
 
-    data:text/html,<body oninput="i.srcdoc=h.value+'<style>'+c.value+'</style><script>'+j.value+'</script>'"><style>textarea,iframe{width:100%;height:50%}body{margin:0}textarea{width:33.33%;font-size:18}</style><textarea placeholder=HTML id=h></textarea><textarea placeholder=CSS id=c></textarea><textarea placeholder=JS id=j></textarea><iframe id=i>
-
-If you want to read the code, check out [index.html](https://github.com/umpox/TinyEditor/blob/master/index.html)
+    data:text/html,<style>body{margin:0}[id]{width:33.33%;height:50%}#i{width: 100%}</style><body oninput="i.srcdoc=h.value+'<style>'+c.value+'</style><script>'+j.value+'</script>'"><textarea placeholder=HTML id=h></textarea><textarea placeholder=CSS id=c></textarea><textarea placeholder=JS id=j></textarea><iframe id=i>
+    
+If you want to read the code, check out [index.html](https://github.com/ZweiZhao/TinyEditor/blob/master/index.html)
```

Comment:
why not make it relative? `[index.html](./index.html)`

Reply:
`./index.html is not 404 page?

Reply:
thx, i have fixed.